third_party/openvino/ngraph_c_api/src/ngraph_c_api.cpp

```diff
@@ -704,4 +726,50 @@ IEStatusCode ngraph_group_convolution_backprop_data(
   CREATE_NODE_AND_CATCH_EXCEPTIONS(conv2d, node);
 }
 
+IEStatusCode ngraph_split(const ngraph_node_t* input,
+                          const ngraph_node_t* axis,
+                          size_t num_splits,
+                          ngraph_node_t** node) {
+  TRY_IE_EXCEPTIONS
+  auto split = std::make_shared<ngraph::op::v1::Split>(
+      input->object, axis->object, num_splits);
+  CREATE_NODE_AND_CATCH_EXCEPTIONS(split, node);
+}
+
+IEStatusCode ngraph_variadic_split(const ngraph_node_t* input,
+                                   const ngraph_node_t* axis,
+                                   const ngraph_node_t* splits,
+                                   ngraph_node_t** node) {
+  TRY_IE_EXCEPTIONS
+  auto split = std::make_shared<ngraph::op::v1::VariadicSplit>(
+      input->object, axis->object, splits->object);
+  CREATE_NODE_AND_CATCH_EXCEPTIONS(split, node);
+}
+
+IEStatusCode ngraph_gru_sequence(const ngraph_node_t* input,
+                                 const ngraph_node_t* initial_hidden_state,
+                                 const ngraph_node_t* sequence_lengths,
+                                 const ngraph_node_t* weight,
+                                 const ngraph_node_t* recurrent_weight,
+                                 const ngraph_node_t* bias,
+                                 const size_t hidden_size,
+                                 ngraph_recurrent_sequence_direction direction,
+                                 ngraph_gru_activation reset_activation,
```

Comment:
ngraph is use std::string, so maybe we need to make it as `char*`.

Reply:
This is a good suggestion, I will use char* arrays and delete enum "ngraph_gru_activation".